src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Shapes/Shapes_Tests.cs

```diff
@@ -46,6 +46,28 @@ void TabWaitAndThenScreenshot(string buttonName)
 			}
 		}
 
+		[Test]
+		[AutoRetry]
+		public void Affect_Measurement_polygon()
+		{
+			Run("SamplesApp.Windows_UI_Xaml_Shapes.PolygonPage");
+			_app.WaitForElement("DPolygon");
+			_app.Marked("ClearShape").Tap();
+			TakeScreenshot($"PolygonPage - ClearShape");
+			Draw_polygon();
```

Comment:
why are you calling another test within this test?

Reply:
It was because i've cleared the point collection and wanted a new one again.
If's not good practice i will create again a draw shape after clear

Reply:
I don't think it's good practice to call one UI test inside another UI test